lib/private/L10N/Factory.php

```diff
@@ -444,12 +486,8 @@ private function getLanguageFromRequest($app) {
 	/**
 	 * if default language is set to de_DE (formal German) this should be
 	 * preferred to 'de' (non-formal German) if possible
-	 *
-	 * @param string|null $app
-	 * @param string $lang
-	 * @return string
 	 */
-	protected function respectDefaultLanguage($app, $lang) {
+	protected function respectDefaultLanguage(?string $app, ?string $lang): string {
```

Comment:
InvalidNullableReturnType

Error
The declared return type 'string' for OC\L10N\Factory::respectDefaultLanguage is not nullable, but 'null|string' contains null

```suggestion
	protected function respectDefaultLanguage(?string $app, string $lang): string {
```